app/controllers/catalog_controller.rb

```diff
@@ -13,14 +13,14 @@ class CatalogController < ApplicationController
   def notify_index_read_only
     return unless Figgy.index_read_only?
     message = ["Figgy is currently undergoing maintenance and resource ingest and editing is disabled."]
-    message << flash[:notice] if flash[:notice]
+    message << flash[:notice] if flash[:notice] && flash[:notice] != message.first
```

Comment:
Okay this seems good but maybe we could try `&& !flash[:notice].starts_with?(message.first)` -- I think that would be a bit more flexible.